eth/v1alpha1/validator.proto

```diff
@@ -48,6 +48,17 @@ service BeaconNodeValidator {
         };
     }
 
+    // Stream validator duties for the requested validators.
+    //
+    // The duties consist of:
+    //   Proposer - the validator that creates a beacon chain block.
+    //   Attester — a validator that is part of a committee that needs to sign off on a beacon chain
+    rpc StreamDuties(stream DutiesRequest) returns (stream DutiesResponse) {
+        option (google.api.http) = {
+            get: "/eth/v1alpha1/validator/duties/stream"
```

Comment:
Bi-directional streaming is not supported for grpc / json. Please remove this option.